### PR TITLE
[Java] Update templates not to add @Pattern on byte array property

### PR DESCRIPTION
**Pull request by beytun**
This fixes the error when validating a bean which contains _byte[]_ typed property.  Without this change below exception is thrown because _Pattern_ validator cannot be used with a byte array.

```java
javax.validation.UnexpectedTypeException: HV000030: No validator could be found for constraint 'javax.validation.constraints.Pattern' validating type 'byte[]'
```

### Change list
- Fix beanValidationCore.mustache templates not to have @Pattern on byte array properties
- Re-generate samples

### PR checklist
 
- [X] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [X] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [X] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) beforehand.
- [X] Run the shell script `./bin/generate-samples.sh`to update all Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. These must match the expectations made by your contribution. You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [X] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`
- [X] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

@bbdouglas @sreeshas @jfiala @lukoyanov @cbornet @jeff9finger @karismann @Zomzog @lwlee2608 @bkabrda

### CODE REVIEW

File: samples/client/petstore/typescript-angular-v10-provided-in-root/builds/default/.openapi-generator/FILES [hunk 1]
```diff
@@ -9,11 +9,11 @@ configuration.ts
...
-model/./order.ts
-model/./pet.ts
-model/./tag.ts
-model/./user.ts
+model/apiResponse.ts
+model/category.ts
+model/order.ts
+model/pet.ts
+model/tag.ts
+model/user.ts
```
Comment (wing328) [inline samples/client/petstore/typescript-angular-v10-provided-in-root/builds/default/.openapi-generator/FILES:17]: @beytun thanks for the PR. Just curious, are you using Git Bash on Windows to update the samples?

Comment (beytun) [reply samples/client/petstore/typescript-angular-v10-provided-in-root/builds/default/.openapi-generator/FILES:17]: Nope, I run the script in zsh on Mac.

Comment (beytun) [reply samples/client/petstore/typescript-angular-v10-provided-in-root/builds/default/.openapi-generator/FILES:17]: I see that "ci/circleci: build" failed, but don't know the issue. Can anyone guide me if I need to do anything?

Comment (borsch) [reply samples/client/petstore/typescript-angular-v10-provided-in-root/builds/default/.openapi-generator/FILES:17]: @beytun CircleCI generate code by it's own and try to match yours changes and his own changes. In the given file you have changed separator(from `./` to `/`) which doesn't match to what is generated in CircleCI

Comment (beytun) [reply samples/client/petstore/typescript-angular-v10-provided-in-root/builds/default/.openapi-generator/FILES]: Thanks, @borsch. After your comment, I tried other shells to generate code but the result is some. So I manually changed those files back to original state. Now CircleCI is also successful.

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

**Comment by beytun** [comment]
> Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.
> 
> Let me know if you need help fixing it.
> 
> Ref: https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

Thanks @wing328. Amended commits.

**Comment by beytun** [comment]
Hi @wing328. Can this be backported to 4.x version?

**Pull request merged**